media_player/alexa.py

```diff
@@ -417,11 +425,26 @@ def _get_source_list(self):
                 sources.append(devices['friendlyName'])
         return ['Local Speaker'] + sources
 
+    def _get_last_device_serial(self):
```

Comment:
No need for a helper method that gets a value, then simply returns that same value. Just set it on line 338